.github/workflows/native.yml

```diff
@@ -0,0 +1,223 @@
+name: default
+'on':
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+env:
+  ccache_base_dir: ${{ github.workspace }}
+  ccache_dir: "${{ github.workspace }}/.ccache"
+  ccache_compilercheck: content
+  cache_compress: 'true'
+  ccache_maxsize: 50M
```

Comment:
`50M` is sufficient?

Reply:
Recent workflow runs: [Ubuntu 18.04](https://github.com/browsermt/bergamot-translator/pull/181/checks?check_run_id=2714017754), [MacOS 10.15](https://github.com/browsermt/bergamot-translator/pull/181/checks?check_run_id=2714017807) seem to be doing okay, look at the ccache epilog. 


<details>
<summary>Ubuntu 18.04 full</summary>

```
Run ccache -s # Print current cache stats
  ccache -s # Print current cache stats
  shell: /bin/bash -e {0}
  env:
    ccache_base_dir: /home/runner/work/bergamot-translator/bergamot-translator
    ccache_dir: /home/runner/work/bergamot-translator/bergamot-translator/.ccache
    ccache_compilercheck: content
    cache_compress: true
    ccache_maxsize: 50M
    ccache_cmake: -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DCMAKE_C_COMPILER_LAUNCHER=ccache
    CCACHE_COMPILER_CHECK: content
    CCACHE_BASE_DIR: /home/runner/work/bergamot-translator/bergamot-translator
    CCACHE_COMPRESS: 
    CCACHE_DIR: /home/runner/work/bergamot-translator/bergamot-translator/.ccache
    CCACHE_MAXSIZE: 50M
cache directory                     /home/runner/work/bergamot-translator/bergamot-translator/.ccache
primary config                      /home/runner/work/bergamot-translator/bergamot-translator/.ccache/ccache.conf
secondary config      (readonly)    /etc/ccache.conf
stats zero time                     Mon May 31 22:52:01 2021
cache hit (direct)                   221
cache hit (preprocessed)               8
cache miss                            14
cache hit rate                     94.24 %
cleanups performed                     2
files in cache                       700
cache size                          30.1 MB
max cache size                      50.0 MB
```

</details>


<details>
<summary>MacOS 10.15 full</summary>

```
Run ccache -s # Print current cache stats
  ccache -s # Print current cache stats
  shell: /bin/bash -e {0}
  env:
    ccache_base_dir: /Users/runner/work/bergamot-translator/bergamot-translator
    ccache_dir: /Users/runner/work/bergamot-translator/bergamot-translator/.ccache
    ccache_compilercheck: content
    cache_compress: true
    ccache_maxsize: 50M
    ccache_cmake: -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DCMAKE_C_COMPILER_LAUNCHER=ccache
    LDFLAGS: -L/usr/local/opt/openblas/lib
    CPPFLAGS: -I/usr/local/opt/openblas/include
    CCACHE_COMPILER_CHECK: content
    CCACHE_BASE_DIR: /Users/runner/work/bergamot-translator/bergamot-translator
    CCACHE_COMPRESS: 
    CCACHE_DIR: /Users/runner/work/bergamot-translator/bergamot-translator/.ccache
    CCACHE_MAXSIZE: 50M
cache directory                     /Users/runner/work/bergamot-translator/bergamot-translator/.ccache
primary config                      /Users/runner/work/bergamot-translator/bergamot-translator/.ccache/ccache.conf
secondary config (readonly)         /usr/local/Cellar/ccache/4.3/etc/ccache.conf
stats updated                       Mon May 31 23:16:32 2021
stats zeroed                        Mon May 31 23:13:30 2021
cache hit (direct)                   211
cache hit (preprocessed)              14
cache miss                            18
cache hit rate                     92.59 %
cleanups performed                     2
files in cache                       477
cache size                          20.9 MB
max cache size                      50.0 MB

```

</details>

Should I keep more? I'm assuming I have 5GB across all workflows for ccache. Can raise this number potentially depending on how much we blow up matrix.

Reply:
Lets keep it at `50M`